### ISSUE TITLE
Capes

### ISSUE THREAD

**Issue by Emayli** [5zig/The-5zig-Mod#1]
Hello,

So I'm not the best at modifying. Could you help me out on how I can change cape? To illustrate, now that upload feature for capes is gone, I was wondering if you can upload a cape within the source code.

**Comment by freptical** [comment]
How to apply a cape?

**Comment by Craftah** [comment]
Yeah for real how do you add your cape back? I donated in 2017 and have lost my cape now that it doesnt work. Sad this guy dropped his ownership for 5zig :(

**Comment by JSR-Gaming-x** [comment]
The capes relied on servers which have been shutdown by 5zig himself when he quit.
